doc/src/cylc-user-guide/cug.tex

```diff
@@ -394,6 +396,8 @@ \subsection{Third-Party Software Packages}
   \item {\bf pyOpenSSL} - \url{http://www.pyopenssl.org/}
   \item {\bf python-requests} - \url{http://docs.python-requests.org/}
   \item ({\bf python-urllib3} - should be bundled with python-requests)
+  \item ({\bf Python \lstinline@>=@ 2.7.9} (but not yet Python 3) is
+        recommended for the best security.)
```

Comment:
Perhaps merge this into the previous paragraph to make it clear we support Python 2.6 something like:

```tex
{\bf Python \lstinline@>=2.6 <3@} is required {\bf Python \lstinline@>=2.7.9@} is recommended. Python
should already be installed in your Linux system. \url{https://python.org}.
```

Reply:
Done. I have (hopefully) improved the statement regarding Python 2 requirement.